.github/workflows/advisory-cron.yaml

```diff
@@ -4,6 +4,7 @@ on:
     - cron: '0 18 * * *'
 jobs:
   cargo-deny:
+    if: ${{ false }}
```

Comment:
What's the reason for all these conditions? Is this just to check CI runs for deno and then you'll remove them before we merge?

Reply:
Exactly, just wanted to disable any other CI jobs temporarily. I did a test run already and found that the deno test was also being run by the `yarn test` command.

Will reverse these once I can see that the CI runs properly.